ONSdigital/sdc-global-design-patterns
# Question pattern

### What is the context of this PR?
The pattern for question scss exists in the PL but no hbs file or examples can be viewed.

### How to review 
- Check the new question patterns to ensure they function correctly and include all necessary components
- Check the notes to ensure they are easily understood and contain the required information.

### Trello card
https://trello.com/c/4CJ2Los2/35-repeating-question
